crypto/provider_core.c

```diff
@@ -417,9 +417,6 @@ static int provider_activate(OSSL_PROVIDER *prov, OPENSSL_CTX *libctx)
      * same library context as was used for the EVP call that caused this call
      * to "provider_activate".
```

Comment:
The last para of this comment should be removed.

Reply:
Will do

Reply:
Actually, both paragraphs can go, since all the special knowledge about the FIPS module is now gone from here.